### ISSUE TITLE
Portuguese Translation

### ISSUE THREAD

**Issue by D4rkMurd3r** [steverichey/DMLocalizer#13]
A game by: Um jogo de
Powered by: Produzido por 
Press left or right to start: Pressiona esquerda ou direita para começar
Release left and right: Solta esquerda ou direita  
to start: Para começar
to erase: Para apagar
Touch left or right side to start: Toca no lado esquerdo ou direito para começar
Release both sides: Solta ambos os lados
Fill bar to erase save data: Preenche a barra para apagar os dados
Erasing save data: A apagar os dados guardados
attempts: Tentativas
distance: Distância
seconds: Segundos
medals: Medalhas
levels: Níveis
coins: Moedas
trophies: Troféus
zenith: Zenith/ Zénite
zeniths: Zeniths / Zénites
the end: Fim
trophy unlocked: Troféu desbloqueado
paused: Em pausa
time: Tempo
Hold left or right to restart: Pressiona esquerda ou direita para recomeçar
Touch screen to restart: Toca no ecrã para começar

![dont_la](https://cloud.githubusercontent.com/assets/10491395/5698866/5cbf0b0c-9a10-11e4-9f08-145fadf24f1b.png)
![move_la](https://cloud.githubusercontent.com/assets/10491395/5698867/609b1b44-9a10-11e4-9b74-93e605092416.png)
![levelup_la](https://cloud.githubusercontent.com/assets/10491395/5698876/889ac194-9a10-11e4-9db2-1aae3fded35b.png)
![zenith_la](https://cloud.githubusercontent.com/assets/10491395/5698872/6bf2f480-9a10-11e4-94a9-6061f03f4ae3.png)
![gameover_la](https://cloud.githubusercontent.com/assets/10491395/5698873/6ec681f4-9a10-11e4-96a4-3c2986ecfbf6.png)

**Comment by steverichey** [comment]
Excellent, thank you!

**Comment by D4rkMurd3r** [comment]
This is the Portuguese - Portugal translation.

For the credits, my name is Luís Alves.

Thank you!

**Comment by Hirien** [comment]
"Powered by" should be "Desenvolvido com" or "Criado com" instead of "Produzido por" (which actually means "Produced by").

**Comment by D4rkMurd3r** [comment]
If you want to translate it to "Desenvolvido" then it would be "Desenvolvido por" and not "Desenvolvido com" (which actually means "Developed with"). 

Steve you can use what you think it is best. I'll let the literal translation here:

Desenvolvido por - Developed by
Criado por - Created by

Thank you Hirien for the more appropriate word.

**Comment by Hirien** [comment]
**D4rkMurd3r:** When Steve says "Powered by", he means the engine the game was created with. Saying "Produzido por" is wrong because the verb isn't referring to a person. Look:

```
Powered by HaxeFlixel

1) Your translation: Produzido por HaxeFlixel
→  In English: Produced by HaxeFlixel

2) My translation: Desenvolvido/Criado com HaxeFlixel
→  In English: Developed/Created with HaxeFlixel.
```

It's important to note that I'm not only translating the game: I'm localizing it. This means I ought to choose the words that Portuguese-speaking gamers are more comfortable/used with, even if they are not the exact translation of the original ones.

**Comment by steverichey** [comment]
I'm actually dropping "Powered by" for `v1.4`, so don't worry about it. This was also an issue in the German translation! So it's an interesting case of something which does not translate very well from English.

**Comment by D4rkMurd3r** [comment]
I thought it would be his name to show up and not the engine. Sorry Hirien, you're right

**Comment by steverichey** [comment]
Implemented in `0a9017d5`
